css/main.css

```diff
@@ -325,3 +325,32 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+#contact-me {
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+  align-self: center;
+}
+
+#contact-me h2 {
+  color: #172b4d;
+}
+
+#contact-me form {
+  display: flex;
+  flex-direction: column;
+}
+
+#contact-me form * {
+  margin: 15px;
+}
+
+#contact-me form button {
+  background-color: orangered;
+}
```

Comment:
- [ ] Kindly use the same colors as in the Figma design. You can inspect any element and on your right side-bar you should see something like this:
![Captura de Pantalla 2022-09-14 a la(s) 3 57 02 p  m](https://user-images.githubusercontent.com/86022690/190260874-45e6ff44-9a76-4886-a22d-fa03fbc1ba82.png)